### PR TITLE
Added Hadoop Prometheus Dashboard

### PR DESCRIPTION
**Pull request by algchoo**
* added hadoop prometheus dashboard json
* included screenshots
* updated readme and metadata

### CODE REVIEW

File: dashboards/hadoop/README.md [hunk 1]
```diff
@@ -5,8 +5,12 @@
...
 
-
 |Hadoop GCE Overview|
 |:------------------|
 |Filename: [hadoop-gce-overview.json](hadoop-gce-overview.json)|
-|This dashboard has 12 charts for viewing hadoop when monitored by [Google's Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/hadoop#monitored-metrics), including `Data Nodes`, `Volume Failures`, `Capacity Used`, `Capacity Limit`, `Block Count`, `Corrupt Blocks`, `Missing Blocks`, `File Count`, `File Load`, `CPU % Top 5 VMs`, `Memory % Top 5 VMs` and`Hosts By Region`. There is also a `Hadoop Monitoring Link` card with links to docs and hadoop log in Cloud Logging.
+|This dashboard has 12 charts for viewing hadoop when monitored by [Google's Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/hadoop#monitored-metrics), including `Data Nodes`, `Volume Failures`, `Capacity Used`, `Capacity Limit`, `Block Count`, `Corrupt Blocks`, `Missing Blocks`, `File Count`, `File Load`, `CPU % Top 5 VMs`, `Memory % Top 5 VMs` and`Hosts By Region`. There is also a `Hadoop Monitoring Link` card with links to docs and hadoop log in Cloud Logging.|
```
Comment (yqlu) [inline dashboards/hadoop/README.md:11]: Let's capitalize Hadoop

File: dashboards/hadoop/README.md [hunk 2]
```diff
@@ -5,8 +5,12 @@
...
+
+|Hadoop Prometheus Overview|
+|:------------------|
+|Filename: [hadoop-prometheus-overview.json](hadoop-prometheus-overview.json)|
+|This dashboard is based on prometheus metrics exposed by an [exporter](https://github.com/prometheus/jmx_exporter), including `Data Nodes`, `Volume Failures`, `Capacity Used`, `Capacity Limit`, `Block Count`, `Corrupt Blocks`, `Missing Blocks`, `File Count`, and `File Load`.|
```
Comment (yqlu) [inline dashboards/hadoop/README.md:16]: Update exporter URL (and same in metadata.yaml)

**Review by yqlu — CHANGES_REQUESTED** [review]

File: dashboards/hadoop/README.md (hunk 2 above)
Comment (algchoo) [reply dashboards/hadoop/README.md:16]: What would we like it to be updated to?

Comment (yqlu) [reply dashboards/hadoop/README.md:16]: Ah my bad -- I was expecting the URL to be a specialized Hadoop exporter and this was a copy-paste error, but I took a closer look at https://github.com/prometheus/jmx_exporter and figured out what was going on

**Review by yqlu — APPROVED** [review]

**Pull request merged**